app/src/main/java/org/wikipedia/gallery/ImageLicense.kt

```diff
@@ -0,0 +1,66 @@
+package org.wikipedia.gallery
+
+import androidx.annotation.DrawableRes
+import com.google.gson.annotations.SerializedName
+import org.apache.commons.lang3.StringUtils
+import org.wikipedia.R
+import java.io.Serializable
+import java.util.*
+
+class ImageLicense : Serializable {
+
+    @SerializedName("type") val licenseName: String
+
+    @SerializedName("code") val licenseShortName: String
+
+    @SerializedName("url") val licenseUrl: String
+
+    constructor(metadata: ExtMetadata) {
+        licenseName = metadata.license()
+        licenseShortName = metadata.licenseShortName()
+        licenseUrl = metadata.licenseUrl()
+    }
+
+    @JvmOverloads
+    constructor(license: String = "", licenseShortName: String = "", licenseUrl: String = "") {
+        licenseName = license
+        this.licenseShortName = licenseShortName
+        this.licenseUrl = licenseUrl
+    }
+
+    private val isLicenseCC: Boolean
+        get() = (StringUtils.defaultString(licenseName).lowercase(Locale.ENGLISH).startsWith(CREATIVE_COMMONS_PREFIX) || StringUtils.defaultString(licenseShortName).toLowerCase(Locale.ENGLISH).startsWith(CREATIVE_COMMONS_PREFIX))
+    private val isLicensePD: Boolean
+        get() = (StringUtils.defaultString(licenseName).lowercase(Locale.ENGLISH).startsWith(PUBLIC_DOMAIN_PREFIX) || StringUtils.defaultString(licenseShortName).toLowerCase(Locale.ENGLISH).startsWith(PUBLIC_DOMAIN_PREFIX))
+    private val isLicenseCCBySa: Boolean
+        get() = (StringUtils.defaultString(licenseName).lowercase(Locale.ENGLISH).replace("-", "").startsWith(CC_BY_SA) || StringUtils.defaultString(licenseShortName).toLowerCase(Locale.ENGLISH).replace("-", "").startsWith(CC_BY_SA))
```

Comment:
1. `StringUtils.defaultString()` can be removed since the default value of the variables is `""`.
2. Please replace `toLowerCase(Locale.ENGLISH)` to `lowercase(Locale.ENGLISH)`